handlers/workloads.go

```diff
@@ -187,7 +186,7 @@ func PodLogs(w http.ResponseWriter, r *http.Request) {
 	pod := vars["pod"]
 
 	// Get log options
-	opts := businesspkg.LogOptions{PodLogOptions: core_v1.PodLogOptions{Timestamps: true}}
+	opts := business.Workload.BuildLogOptionsCriteria()
```

Comment:
This is good.
But I wonder why wouldn't you like to move the parsing into this method instead to use it just like an initialization.

Look this example in the other handler:

```
criteria := business.ParseIstioConfigCriteria(namespace, objects, labelSelector, workloadSelector)
```

The paremeters are passed into this `ParseIstioConfigCriteria` and then the criteria is used for the business method.
If the parse error fails you can send a back a 500 or 400 (depends if you want to notify user that parameters were incorrect).

Benefit of this is that handler doesn't touch business logic, only invoking business hence you can save to bring the meta_v1 package.

It's just style, this code works, but in rest of the handlers that's the style we used.

In any case this solves the previous request (thank you!).